packages/kit/src/runtime/client/client.js

```diff
@@ -1513,6 +1528,54 @@ export function create_client({ target, base }) {
 				});
 			});
 
+			target.addEventListener('submit', (event) => {
+				if (event.defaultPrevented) return;
+
+				const form = /** @type {HTMLFormElement} */ (
+					HTMLFormElement.prototype.cloneNode.call(event.target)
+				);
+
+				const submitter = /** @type {HTMLButtonElement | HTMLInputElement} */ (event.submitter);
+
+				const method = event.submitter?.hasAttribute('formmethod')
+					? submitter.formMethod
+					: form.method;
```

Comment:
```suggestion
				const method = submitter?.formMethod ?? form.method;
```

would this work?

Reply:
almost. the reason it's like this is that `submitter.formAction` is always defined whether or not there's an `action` attribute; i thought the same thing would be true for `formMethod`. Turns out it's the empty string if the attribute is unspecified. So we should be able to do this (`||` instead of `??`):

```suggestion
				const method = submitter.formMethod || form.method;
```

Reply:
Don't we also need the `cloneNode` thing due to https://github.com/sveltejs/kit/issues/7593? And if we can simplify that, we should also simplify that in `runtime/app/forms.js`